setup/action.yml

```diff
@@ -1,7 +1,10 @@
-name: 'sprkl: setup'
-description: 'Set up sprkl'
+name: "sprkl: setup and push"
+decription: "Setup sprkl and push sprkl snapshot to CI platform at the end of the job"
 
 inputs:
+  token:
+    description: ""
```

Comment:
Add description: "Sprkl allocated token - get your token at by installing the Sprkl GitHub Application"